bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/calc/SunCalc.java

```diff
@@ -144,6 +144,50 @@ private boolean isSunUpAllDay(Calendar calendar, double latitude, double longitu
         return true;
     }
 
+    /**
+     * Returns the time when the sun is at its minimum elevation
+     */
+    private Calendar getMinSunElevationTime(@Nullable Calendar noon, double latitude, double longitude,
+            @Nullable Double altitude) {
+        Objects.requireNonNull(noon, "A solar noon should always exist");
+
+        Calendar cal = (Calendar) noon.clone();
+        Sun sun = new Sun();
+
+        double minElevation = Double.POSITIVE_INFINITY;
+        Calendar minCal = (Calendar) noon.clone();
+
+        Double previousElevation = null;
+        boolean wasDecreasing = false;
+
+        final int MINUTES_SPAN = 14 * 60; // don't scan 24h, 14 will be enough
```

Comment:
This confuses me a bit - 14 hours will be "enough" if you make sure that you scan the correct part of the cycle? Or, what is the idea? Because, with time zones, the lowest elevation can actually be any clock time. I'm not sure exactly how this is applies/used, especially since the value isn't actually used for anything.